src/Controls/src/Core/Compatibility/Handlers/FlyoutPage/iOS/PhoneFlyoutPageRenderer.cs

```diff
@@ -476,6 +478,24 @@ void LayoutChildren(bool animated)
 				UpdateClickOffViewFrame();
 		}
 
+		void UpdateFlyoutLayoutBehaviorChanges()
+		{
+			LayoutChildren(true);
+            FlyoutPage flyoutPage = (FlyoutPage)Element;
```

Comment:
```
FlyoutPage flyoutPage = Element as FlyoutPage;
if (flyoutPage == null) return;
```

Reply:
Based on your suggestion, I’ve made the changes. Let me know if any further modifications are needed.